src/lib/components/WizardModal.svelte

```diff
@@ -3,47 +3,62 @@
   import WizardTransition from "./WizardTransition.svelte";
   import { WizardStepsState } from "$lib/stores/wizard.state";
   import type { WizardStep, WizardSteps } from "$lib/types/wizard";
-  import { createEventDispatcher } from "svelte";
+  import type { Snippet } from "svelte";
 
-  // eslint-disable-next-line no-undef -- The still linter does not understand the Svelte generics type
-  export let steps: WizardSteps<T>;
-  export let disablePointerEvents = false;
-  export let testId: string | undefined = undefined;
+  interface Props {
+    // eslint-disable-next-line no-undef -- The still linter does not understand the Svelte generics type
+    steps: WizardSteps<T>;
```

Comment:
I wonder if they don't already use a pattern for generic NNS dapp (without linter). Have you had a look?

Reply:
Can we use `generics`? https://svelte.dev/docs/svelte/typescript#Generic-$props

Reply:
Sounds good, thx